code/auto_download/hospitalisations-download.R

```diff
@@ -0,0 +1,16 @@
+# Download and save all hospitalisations data
+#  plus save a "truth" file with only the selected source for each country
+
+library(here)
+script_dir <- here("code", "auto_download", "hospitalisations")
+
+# Download and save minimally processed data files
+source(here(script_dir, "get-ecdc-official.R"))
```

Comment:
Should this (and the next few lines) be `file.path` instead of `here` as line 5 has already added the necessary folder structure? I presume you've tested it and it works in which case I've learned something.

Reply:
I think `here` handles this with its own magic (so I just use `here` everywhere). In this case
```
> file.path(script_dir, "get-ecdc-official.R") == here(script_dir, "get-ecdc-official.R")
[1] TRUE
```